examples/diviner/python_env.yaml

```diff
@@ -2,4 +2,7 @@ build_dependencies:
   - pip
 dependencies:
   - diviner
+  # cmdstanpy 1.3.0 causes AttributeError in prophet 1.2.0: 'Prophet' object has no attribute 'stan_backend':
+  # https://github.com/facebook/prophet/issues/2697
+  - cmdstanpy<1.3.0
```

Comment:
Hey Cladue,

- Move this constraint in requirements/constraints.txt
- Remove the same constraint in examples/prophet/python_env.yaml
- Add PIP_CONSTRAINT in examples.yml (see master.yml) as an example